### ISSUE TITLE
Ideas for frontend side configuration

### ISSUE THREAD

**Issue by tarlepp** [UniBot/UniBot#42]
It would be nice if frontend side also have possible for some configurated values. There values could be like date format. Any ideas how to pass some configuration values to frontend side?

**Comment by ProLoser** [comment]
That isn't too hard, I made an endpoint that gives the UniBot version to
the fronted from the config, we can add it there. If you are taking about
channel specific configurations, the plug-ins are sort of supposed to
supply whatever data they want and then use that data in their supplied
view, so that shouldn't be too difficult.

**Comment by tarlepp** [comment]
Yep, something like that should work like a charm. Perhaps that plugin controller should ask directly if itself has some configuration values specified on it.

This won't work on all the cases but, eg. when you have plugin with ```default``` config and no override configuration on ```config.js``` file. This is something that should be take care of.
